### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by JelilFaisalAbudu**
# Requesting code review for the To-Do-List project

* My partner I have finished working on this project and it's now ready for your review.

## Features Implemented
- User can create multiple project, each with several tasks
- User can delete a prject with all its tasks
- User can also remove individual tasks from a project
- User can edit each task
- There is a checkbox to show indicate a completed task
- Each completed task is subtacted from the remaining tasks to be completed.

### CODE REVIEW

File: src/modules/todoLogics.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    const dueDateElement = toDoElement.querySelector('.due-date');
+    dueDateElement.innerText = todo.dueDate;
+
+    const priorityElement = toDoElement.querySelector('.priority');
+    priorityElement.innerText = todo.priority;
+
+
+    const checkbox = toDoElement.querySelector('.checkbox');
+    // checkbox.id = todo.id;
```
Comment (mosaaleb) [inline src/modules/todoLogics.js]: - [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  e.preventDefault();
+  const projectTitle = myVar.newProjectTitleInput.value;
+
+  if (projectTitle === null || projectTitle === '') {
+    return;
+  }
+  const newProject = createProject(projectTitle);
+  myVar.newProjectTitleInput.value = null;
+  myVar.projectList.push(newProject);
+  saveAndRender();
```
Comment (mosaaleb) [inline src/index.js:37]: - [ ] **[OPTIONAL]** Let's hide the modal after successful project submission for a better user experience.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+});
+
+
+myVar.newTodoForm.addEventListener('submit', (e) => {
```
Comment (mosaaleb) [inline src/index.js:41]: - [ ] **[OPTIONAL]** Same here for hiding todo modal after successful submission. Also, the same when you **update** a todo item

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Well done @eypsrcnuygr @JelilFaisalAbudu :clap:. Most requirements are handled, yet there are some minor requirements to handle before moving on.


- [x] There should be a default project which combines all todos from all projects




- [ ] When I first load the application (no projects are added yet), I got this error:

Screenshot from 2020-11-12 18-20-17

Let's fix this.


Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

**Comment by JelilFaisalAbudu** [comment]
# Requesting a second code review for **to-do list** project

- The changes requested by the previous TSE have been done. The project is therefore ready for another review.

## Required changes made

- A default project is added.
- Console error resolved
- Modals automaticaaly disappear after submission

File: src/index.js (hunk 3 above)
Comment (Srjnnnn) [reply src/index.js:41]: Modals are disappearing after submission.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**